apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ConsumerController.java

```diff
@@ -119,6 +120,16 @@ public List<ConsumerRole> assignNamespaceRoleToConsumer(@PathVariable String tok
     return consumerService.assignNamespaceRoleToConsumer(token, appId, namespaceName);
   }
 
+  @GetMapping("/consumers/list")
```

Comment:
iMaintain uniform URL specifications  will be more cleaner.

/consumers/list => /consumers?

Reply:
Will this conflict with @PostMapping(value = "/consumers")? I'm not sure if the definition of createConsumer needs to be modified.

Reply:
> Will this conflict with @PostMapping(value = "/consumers")? 

That's all right as each path can have multiple HTTP methods against it.

Reply:
The conflict here refers to the conflict of method semantics, not the conflict of method mapping. However, it may also be my own understanding of the problem

Reply:
Method name is the part of endpoint semantics by restful standard. So that's all right~